### ISSUE TITLE
Importing into STS shows 'no codesets'.

### ISSUE THREAD

**Issue by kdvolder** [spring-guides/gs-spring-boot-docker#9]
This because of this:
https://github.com/spring-guides/gs-spring-boot-docker/blob/master/.codesets.json#L1

The 'codesets.json' file like this tells STS there is nothing of interest in this guide to import.

Is that intentional? If so okay. Otherwise probably you want to delete the '.codesets.json' file so STS will be able to import the 'initial' and 'complete' versions of this guide as maven/gradle projects.

**Comment by dsyer** [comment]
Pretty sure that was a mistake (since I have no idea what a code set is). Just delete it?

**Comment by kdvolder** [comment]
A 'codeset' is some folder in the github repo that can be checked-out by STS as a project in the workspace, by the 'Import Getting Started Content' wizard. So for most guides this is the 'initial' and 'complete' state of the project. If the .codesets file is removed then STS will assume this as the default.

But there are some special cases, for example where a few guides do not have anything to import. 

So long story short, yes, delete it. Because it looks like this guide is following the default format.

**Comment by kdvolder** [comment]
For some info on this mystical file, have peek at the readme of this project:
https://github.com/kdvolder/gs-mock-fortesting
